.github/workflows/ci.yml

```
@@ -1,24 +1,26 @@
 name: ci
 
-on: ["pull_request", "push"]
+on: ["pull_request"]
```

This should move the push to only run _after_ the merge.


```suggestion
on: ["pull_request"]
  push:
    branches:
      - 'main'
```